config/routes.rb

```diff
@@ -11,15 +11,26 @@
     end
   end
 
-  resources :organizations, only: [:index, :show, :edit, :create, :update, :new]
+  resources :organizations, only: [:index, :show, :edit, :create, :update, :new] do
+    resources :memberships do
```

Comment:
Also, a question. Do we need a memberships resource at all here? I would propose that the action to join an organization would be

```
/organizations/:organization_id/join
```

and therefore you would do something like

```ruby
resources :organizations, only: [...] do
  post :join, to: 'memberships#create`, on: :member
end
```

Thoughts?